### ISSUE TITLE
When a page is sent to email it is cut off just around 1200 characters.

### ISSUE THREAD

**Issue by drmartin66** [rNeomy/reader-view#171]
When clicking on the "Email the contents" button, an email is openned but if it is a long page/article only the first 1200 or so characters are included.  The original page link does appear at the bottom of the email.

Only tested on Gmail with Chrome Version 113.0.5672.127 (Official Build) (64-bit)
Reader View 0.7.1

Tested with the following pages.

https://www.theguardian.com/books/2023/may/20/m-john-harrison-i-want-to-be-the-first-human-to-imitate-chatgpt-wish-i-was-here

https://www.theringer.com/succession/2023/5/31/23743725/succession-finale-writing-jesse-armstrong-scripts-hbo?src=longreads

**Comment by rNeomy** [comment]
True, the extension generates `mailto:` link to send an email. There is a limit on a link length in each browser. So we cannot send the complete content. We need to trim it first.
